store/sgxleveldb/sgx.go

```diff
@@ -0,0 +1,251 @@
+// Package leveldb implements a light client level db for panacea-doracle.
+// It does include Set & Get functions that are sealed & unsealed in the sgx environment.
+
+package sgxleveldb
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/medibloc/panacea-doracle/sgx"
+	log "github.com/sirupsen/logrus"
+	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/errors"
+	"github.com/syndtr/goleveldb/leveldb/opt"
+	"github.com/syndtr/goleveldb/leveldb/util"
+	tmdb "github.com/tendermint/tm-db"
+)
+
+var (
+	// errBatchClosed is returned when a closed or written batch is used.
+	errBatchClosed = errors.New("batch has been written or closed")
+
+	// errKeyEmpty is returned when attempting to use an empty or nil key.
+	errKeyEmpty = errors.New("key cannot be empty")
+
+	// errValueNil is returned when attempting to set a nil value.
+	errValueNil = errors.New("value cannot be nil")
+)
+
+type dbCreator func(name string, dir string) (tmdb.DB, error)
+
+var backends = map[tmdb.BackendType]dbCreator{}
+
+func registerDBCreator(backend tmdb.BackendType, creator dbCreator, force bool) {
+	_, ok := backends[backend]
+	if !force && ok {
+		return
+	}
+	backends[backend] = creator
+}
```

Comment:
It seems that the `backends` map isn't used anywhere after being initialized.

Reply:
This topic has been moved to https://github.com/medibloc/panacea-doracle/pull/12#issuecomment-1223638247